package.json

```diff
@@ -4,7 +4,7 @@
     "jsrsasign": "^8.0.12"
   },
   "name": "@scalar-labs/scalardl-javascript-sdk-base",
-  "version": "1.3.1",
+  "version": "2.0.0",
```

Comment:
I'm not sure if it should be `2.0.0`. Do the existing APIs won't work from these change ?
Our rule is 
major.minor.bugfix and major will be incremented if APIs are changed and minor will be incremented if some features are added without breaking the API.

Reply:
The major changes is because at `constructor(ledgerClient, protobuf, properties)`, it has changed to `constructor(service, protobuf, properties)` where `service` is now an object of `ledgerClient` and `ledgerPrivileged`. Or maybe we could consider it to be minor changes?

Reply:
OK, let's keep it as it is (2.0.0).